cmd/oras/internal/option/format.go

```diff
@@ -56,9 +56,10 @@ var (
 		Usage:     "Print output using the given Go template",
 		HasParams: true,
 	}
+	// Deprecated: table format is deprecated and will be removed in the future.
```

Comment:
We can remove this comment as this is not a library code.

Reply:
Removed.